src/main/scala/mesosphere/chaos/example/Main.scala

```diff
@@ -1,23 +1,21 @@
 package mesosphere.chaos.example
 
 import org.rogach.scallop.ScallopConf
-import mesosphere.chaos.http.{ HttpService, HttpConf, HttpModule }
+import mesosphere.chaos.http.{HttpConf, HttpModule}
 import mesosphere.chaos.metrics.MetricsModule
 import mesosphere.chaos.{ App, AppConfiguration }
 
+// TODO: put the whole example somewhere else
 object Main extends App {
-
-  //Declare all Guice Modules
-  def modules() = {
-    Seq(
-      new HttpModule(conf),
-      new MetricsModule,
-      new ExampleRestModule)
-  }
-
   //The fact that this is lazy, allows us to pass it to a Module
   //constructor.
   lazy val conf = new ScallopConf(args) with HttpConf with AppConfiguration
+  conf.afterInit()
 
-  run(classOf[HttpService])
+  lazy val metricRegistry = new MetricsModule(conf).registry
```

Comment:
@aquamatthias I totally agree on that. Though I am not sure if the package structure should look like `mesosphere.chaos.chaos.ChaosModule` or `mesosphere.chaos.ChaosModule`, last one containing all other module packages like `mesosphere.chaos.http` or `mesosphere.chaos.rest`. Being uncreative with naming as I am right now, I am going with `mesosphere.chaos.chaos.ChaosModule` version. Let me know if I should do different.